tests/unit/test_feedback.py

```diff
@@ -64,10 +64,7 @@ def test_skipeval_all(self) -> None:
 
         self.assertIsInstance(res.result, float)
 
-        self.assertIs(
-            res.result, np.nan
-        )  # NOTE: cannot use assertEqual for nans.
-        # Result should be nan if all evals were skipped.
+        assert np.isnan(res.result)
```

Comment:
can you say `self.assertTrue(np.isnan(res.result))` instead? Technically, the `assert` can be disabled.